client.go

```diff
@@ -299,10 +307,7 @@ func (client *Client) eventFromMessage(message string, level Level) *Event {
 
 func (client *Client) eventFromException(exception error, level Level) *Event {
 	if exception == nil {
-		event := NewEvent()
-		event.Level = level
-		event.Message = fmt.Sprintf("Called %s with nil value", callerFunctionName())
-		return event
+		exception = usageError{fmt.Errorf("%s called with nil error", callerFunctionName())}
```

Comment:
Moved the function name to the very beginning because it is the most relevant information.